Jenkinsfile

```diff
@@ -55,6 +55,8 @@ pipeline {
                         error('Bad input, nothing to do.')
                     }
 
+                    currentBuild.displayName = "${REPO_FULL_NAME}:${PR_ID}"
```

Comment:
We could use `setBuildNameFromArtifactId(artifactId: artifactId)` (but artifactId is defined on the next line).

Reply:
updated